webviz_subsurface/containers/_parameter_response_correlation.py

```diff
@@ -0,0 +1,622 @@
+from uuid import uuid4
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from plotly.subplots import make_subplots
+from dash.exceptions import PreventUpdate
+from dash.dependencies import Input, Output
+import dash_html_components as html
+import dash_core_components as dcc
+import webviz_core_components as wcc
+from webviz_config.webviz_store import webvizstore
+from webviz_config.common_cache import CACHE
+from webviz_config import WebvizContainerABC
+
+from .._datainput.fmu_input import load_parameters, load_csv
+
+
+class ParameterResponseCorrelation(WebvizContainerABC):
+    """### Parameter and response correlation
+
+This container shows correlation between numerical input parameters and responses.
+
+Input can be given either as:
+
+- Aggregated csv files for parameters and responses,
+- An ensemble name defined in shared_settings and a local csv file for responses
+stored per realizations.
```

Comment:
- Remove empty line 25, and instead indent the two lines starting with `-` for readability?
- `shared_settings` instead of shared_settings.